portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <h1>My Hidden Talents!</h1>
+    <div id="content">
```

Comment:
FYI: generally we keep IDs unique https://www.w3schools.com/html/html_id.asp and classes are applied to many elements :)

Reply:
I like the separation of concerns between structure and presentation, though!

Reply:
Thank you so much Bill! I will admit I did not know the difference. I will go back and change the ids which I used multiple times to classes.

Reply:
Sure! Yeah interestingly the styles and everything still work. I suppose it might help to identify that some are special elements (id) and others are just one of many of this "type" (class).

Reply:
Thank you! It's nice when names and types can be informative like that. (Surprisingly, though I am reading that 'classes' don't have the usual properties of those in OOP)